Automattic/newspack-plugin
# feat: remove laterpay

### All Submissions:






### Changes proposed in this Pull Request:



LaterPay has developed a new version of their product, which is not currently AMP compatible. Therefore, removing support for the time being.

Closes https://github.com/Automattic/newspack-plugin/issues/540

### How to test the changes in this Pull Request:

1. Navigate to Reader Revenue wizard, observe LaterPay UI has been removed.
2. Navigate to Plugins dashboard, observe Laterpay is no longer listed as a supported plugin.

### Other information:
